packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -97,322 +381,6 @@
-					label: 'Admin Panel',
-					icon: 'cloud',
-					available: this.settingsStore.isCloudDeployment && hasPermission(['instanceOwner']),
-				},
-				{
-					// Link to in-app templates, available if custom templates are enabled
-					id: 'templates',
-					icon: 'box-open',
-					label: this.$locale.baseText('mainSidebar.templates'),
-					position: 'bottom',
-					available:
-						this.settingsStore.isTemplatesEnabled && this.templatesStore.hasCustomTemplatesHost,
-					route: { to: { name: VIEWS.TEMPLATES } },
-				},
-				{
-					// Link to website templates, available if custom templates are not enabled
-					id: 'templates',
-					icon: 'box-open',
-					label: this.$locale.baseText('mainSidebar.templates'),
-					position: 'bottom',
-					available:
-						this.settingsStore.isTemplatesEnabled && !this.templatesStore.hasCustomTemplatesHost,
-					link: {
-						href: this.templatesStore.websiteTemplateRepositoryURL,
-						target: '_blank',
-					},
-				},
-				{
-					id: 'variables',
-					icon: 'variable',
-					label: this.$locale.baseText('mainSidebar.variables'),
-					customIconSize: 'medium',
-					position: 'bottom',
-					route: { to: { name: VIEWS.VARIABLES } },
-				},
-				{
-					id: 'executions',
-					icon: 'tasks',
-					label: this.$locale.baseText('mainSidebar.executions'),
-					position: 'bottom',
-					route: { to: { name: VIEWS.EXECUTIONS } },
-				},
-				{
-					id: 'help',
-					icon: 'question',
-					label: this.$locale.baseText('mainSidebar.help'),
-					position: 'bottom',
-					children: [
-						{
-							id: 'quickstart',
-							icon: 'video',
-							label: this.$locale.baseText('mainSidebar.helpMenuItems.quickstart'),
-							link: {
-								href: 'https://www.youtube.com/watch?v=1MwSoB0gnM4',
-								target: '_blank',
-							},
-						},
-						{
-							id: 'docs',
-							icon: 'book',
-							label: this.$locale.baseText('mainSidebar.helpMenuItems.documentation'),
-							link: {
-								href: 'https://docs.n8n.io?utm_source=n8n_app&utm_medium=app_sidebar',
-								target: '_blank',
-							},
-						},
-						{
-							id: 'forum',
-							icon: 'users',
-							label: this.$locale.baseText('mainSidebar.helpMenuItems.forum'),
-							link: {
-								href: 'https://community.n8n.io?utm_source=n8n_app&utm_medium=app_sidebar',
-								target: '_blank',
-							},
-						},
-						{
-							id: 'examples',
-							icon: 'graduation-cap',
-							label: this.$locale.baseText('mainSidebar.helpMenuItems.course'),
-							link: {
-								href: 'https://docs.n8n.io/courses/',
-								target: '_blank',
-							},
-						},
-						{
-							id: 'about',
-							icon: 'info',
-							label: this.$locale.baseText('mainSidebar.aboutN8n'),
-							position: 'bottom',
-						},
-					],
-				},
-			];
-			return items;
-		},
-		userIsTrialing(): boolean {
-			return this.cloudPlanStore.userIsTrialing;
-		},
-	},
-	async mounted() {
-		this.basePath = this.rootStore.baseUrl;
-		if (this.$refs.user) {
```

Comment:
How are we referencing the user when the template does not have a ref=user property? 🤔

Reply:
Good catch, looks like the `ref` is not there for a while. Let me check which element is expected here and I'll update